subjects/localhost/audit/README.md

```diff
@@ -0,0 +1,67 @@
+#### Functional
+
+#### Localhost is about program your own HTTP server and test it with an actual browser.
+###### Take The necessary time to understand the project and to test it,take a look around the code to ask reasonable questions such as:
+###### Explain the basics of an HTTP server.
+###### Ask which function they used for I/O Multiplexing.
+###### Ask to get an explanation of how select (or equivalent) is working.
+###### Ask if they use only one select (or equivalent) and how they've managed the server accept and the client read/write.
+###### There should be only one read or one write per client per select (or equivalent). Ask to show you the code that goes from the select (or equivalent) to the read and write of a client.
+###### Search for all [read,recv,write,send] and check if the returned value is well checked. (checking only -1 or 0 is not good you should check both)
+###### Check return value for O/I functions [read,recv,write,send] . is the audite checking for `-1` and `0` at the same time. 
```

Comment:
- By convention it is "I/O" and not "O/I".
- Extra space after "send]".
- Cap missing after the ".".
- audite**e**.
- Second phrase could be more clear by saying "The auditee should check for `-1` and `0` at the same time." (actual version do not say what to expect).